docs/develop/01_sdk/04_integrate/01_nodejs.md

```diff
@@ -6,15 +6,10 @@ title: NodeJS
 import CodeBlock from '@theme/CodeBlock';
 import QueryAccountName from '!!raw-loader!@site/code_examples/core_features/linking/03_account_web3name_query.ts';
 
-
 NodeJS is natively supported and doesn't require any additional setup.
 
 Have a look at these example `package.json` and `index.js` files for reference:
 
-<!--
```

Comment:
This we don't want to dynamically generate anymore?

Reply:
That is just a `package.json`. I updated the CI job to change SDK versions in this folder as well, so that would be enough I feel. I don't see much value in dynamically generating a config file, since it would also then require to spin up a test project and see if it can be created correctly. I felt it is out of scope for this PR, if we want to tackle that at all.